pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -528,6 +532,34 @@ one = 'Format in which to display the Kafka topic (choose from: "json", "yml", "
 [kafka.topic.list.flag.output.description]
 one = 'Format in which to display the Kafka topic (choose from: "json", "yml", "yaml")'
 
+[kafka.topic.produce.flag.key.description]
+one = 'The key associated with the value produced. Empty if not set'
+
+[kafka.topic.produce.flag.key.file]
+one = 'Path to file containing value'
+
+[kafka.topic.produce.flag.partition.description]
+one = 'The partition receiving the message'
+
+[kafka.topic.produce.info.produceSuccess]
+description = 'Produce success message'
+one = 'Record successfully produced to "{{.Topic}}"'
+
+[kafka.topic.consume.flag.partition.description]
+one = 'The partition to consume from'
+
+[kafka.topic.consume.flag.timestamp.description]
+one = 'Timestamp to start consuming from'
```

Comment:
We spend a lot of effort tracking timestamp values that backend support.
We should be as specific as possible what timestamp is